frontend/packages/console-dynamic-plugin-sdk/src/index.ts

```diff
@@ -1,5 +1,6 @@
 // Plugin APIs and types
 export * from './api/useResolvedExtensions';
+export * from './api/useActiveNamespace';
```

Comment:
This needs to be exposed through the `api`.
cc @vojtechszocs

Reply:
Right, this index right now is used mainly by Console monorepo packages.

What's transpiled & distributed to consuming plugin is configured via `files` in SDK's TS config.